beekeeper-cleanup/src/main/java/com/expediagroup/beekeeper/cleanup/context/CommonBeans.java

```diff
@@ -76,14 +85,43 @@ public S3BytesDeletedReporter s3BytesDeletedReporter(S3Client s3Client, MeterReg
   }
 
   @Bean
-  public PathCleaner pathCleaner(S3Client s3Client, S3BytesDeletedReporter s3BytesDeletedReporter) {
+  public S3PathCleaner s3PathCleaner(S3Client s3Client, S3BytesDeletedReporter s3BytesDeletedReporter) {
     return new S3PathCleaner(s3Client, new S3SentinelFilesCleaner(s3Client), s3BytesDeletedReporter);
   }
 
   @Bean
-  public CleanupService cleanupService(HousekeepingPathRepository repository, PathCleaner pathCleaner,
-      @Value("${properties.cleanup-page-size}") int pageSize,
-      @Value("${properties.dry-run-enabled}") boolean dryRunEnabled) {
-    return new PagingCleanupService(repository, pathCleaner, pageSize, dryRunEnabled);
+  public HiveClient hiveClient(
+      @Value("${properties.dry-run-enabled}") boolean dryRunEnabled
+  ) throws MetaException {
+    HiveMetaStoreClient hiveMetaStoreClient = new HiveMetaStoreClient(new HiveConf());
+    return new HiveClient(hiveMetaStoreClient, dryRunEnabled);
+  }
+
+  @Bean
+  public HivePathCleaner hivePathCleaner(
+      HiveClient hiveClient
+  ) {
+    return new HivePathCleaner(hiveClient);
+  }
+
+  @Bean
+  public EnumMap<LifeCycleEventType, PathCleaner> pathCleanerMap(
```

Comment:
This `EnumMap` looks like a factory and is pretty much acting as one, so perhaps we should just have a factory bean instead?

Reply:
Replaced w/ an autowired list that populates all components of `MessageEventMapper`

Reply:
🤦‍♂  - misread this filename and thought this was in `beekeeper-path-scheduler-apiary`. The equivalent has been done there - will need to update this when I switch gears to the cleanup side